quetz/tasks/mirror.py

```diff
@@ -292,30 +329,17 @@ def initial_sync_mirror(
             else:
                 logger.debug(f"updating package {package_name} form {arch}")
 
-            try:
-                remote_package = remote_repository.open(path)
-            except RemoteServerError:
-                logger.error(f"remote server error when getting a file {path}")
-                continue
+            update_batch.append(path)
+            update_size += metadata['size']
```

Comment:
you need to add the size to fixtures

Reply:
I've added a default value for now. That might be more robust anyways?